src/main/java/org/jabref/logic/citationstyle/CSLAdapter.java

```diff
@@ -84,13 +85,22 @@ private void initialize(String newStyle, CitationStyleOutputFormat newFormat) th
          * Converts the {@link BibEntry} into {@link CSLItemData}.
          */
         private static CSLItemData bibEntryToCSLItemData(BibEntry bibEntry) {
+
+            // create a copy of bibEntry
+            bibEntry = (BibEntry) bibEntry.clone();
+
             String citeKey = bibEntry.getCiteKeyOptional().orElse("");
             BibTeXEntry bibTeXEntry = new BibTeXEntry(new Key(bibEntry.getType()), new Key(citeKey));
 
             // Not every field is already generated into latex free fields
             HTMLChars latexToHtmlConverter = new HTMLChars();
             RemoveNewlinesFormatter removeNewlinesFormatter = new RemoveNewlinesFormatter();
             for (String key : bibEntry.getFieldMap().keySet()) {
+                if (FieldName.MONTH.equals(key)) {
+                    // change month field value from "#mon#" to "mon"
```

Comment:
I don't understand why this hack is necessary at this place.

In case a field value is surrounded by `#`, JabRef stores tha value as follows:

    month = mar

and not

    month = #mar#

See http://help.jabref.org/en/EntryEditor#field-consistency-checking for an explanation. So, I think, either the wrong method is used to get the month or there is something more wrong on JabRef: All other exports should have difficulties.

Reply:
This is actually necessary. The problem is that csl can not handle the JabRef special Syntax with the #. Therefore the value of the month field without the # is needed.

For export usually a latex formatter is applied which takes care of this

Reply:
@koppor As CSL is not part of our code we cannot inject a instructions that handle #mon# forma.
The issue on CSL plugin is still open [Bibtex month like #mar# not parsed #41](https://github.com/michel-kraemer/citeproc-java/issues/41)